src/System.Memory/src/System/Span.cs

```diff
@@ -424,8 +424,8 @@ public T[] ToArray()
         //   _pinnable   = null
         //   _byteOffset = the pointer
         //
-        private readonly Pinnable<T> _pinnable;
-        private readonly IntPtr _byteOffset;
-        private readonly int _length;
+        internal readonly Pinnable<T> _pinnable;
+        internal readonly IntPtr _byteOffset;
+        internal readonly int _length;
```

Comment:
We have to use the raw Ctor since DangerousCreate() doesn't have the right signature, but exposing the raw Ctor is far less odious than exposing the fields. Thanks!